### PR TITLE
Directory Option added "-d"

### PR DESCRIPTION
**Pull request by franzNJ-18**
Option added to Directory Class

### CODE REVIEW

File: src/main/java/cmd/SimpleCmd.java [hunk 1]
```diff
@@ -60,6 +62,19 @@ public static void main(String... args) {
    */
   public static File getCurrentLocation() {
 
+    return getLocation();
```
Comment (fzechert-cg) [inline src/main/java/cmd/SimpleCmd.java:65]: I think this is not needed.

File: src/main/java/cmd/commands/dir/DirCommand.java [hunk 2]
```diff
@@ -50,6 +57,24 @@ private void listFilesInDirectory(File directory) {
         }
     }
 
+//    public void setDirectory(String path) {
```
Comment (fzechert-cg) [inline src/main/java/cmd/commands/dir/DirCommand.java:60]: I think this is not needed.

**Review by fzechert-cg — CHANGES_REQUESTED** [review]

File: src/main/java/cmd/SimpleCmd.java (hunk 1 above)
Comment (franzNJ-18) [reply src/main/java/cmd/SimpleCmd.java:65]: Yes, that is not needed

File: src/main/java/cmd/commands/dir/DirCommand.java (hunk 2 above)
Comment (franzNJ-18) [reply src/main/java/cmd/commands/dir/DirCommand.java:60]: Yes, that is not needed

**Pull request closed without merging**